.github/workflows/generate_diffs.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#on:
+#  workflow_dispatch:
+#  schedule:
+#  # every morning
+#  - cron: '0 1 * * *'
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+name: monorepo-diff
+jobs:
+  generate-diffs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        repository: 'googleapis/google-cloud-java'
+    - name: Run generate-diff script
+      run: ./generation/generate_diff.sh
+      env:
+        USERNAME: ${{ github.actor }}
```

Comment:
Did it work? If not `YOSHI_CODE_BOT_TOKEN` is available.

https://github.com/googleapis/google-cloud-java/blob/main/.github/workflows/googleapis_hermetic_sync.yaml#L41

Reply:
Yep, https://github.com/googleapis/google-cloud-java/pull/8448 and https://github.com/googleapis/google-cloud-java/pull/8449 both have `github-actions force-pushed...`